src/components/ha-selector/ha-selector-time.ts

```diff
@@ -0,0 +1,46 @@
+import { customElement, html, LitElement, property } from "lit-element";
+import { HomeAssistant } from "../../types";
+import { TimeSelector } from "../../data/selector";
+import { fireEvent } from "../../common/dom/fire_event";
+import "../paper-time-input";
+
+@customElement("ha-selector-time")
+export class HaTimeSelector extends LitElement {
+  @property() public hass!: HomeAssistant;
+
+  @property() public selector!: TimeSelector;
+
+  @property() public value?: string;
+
+  @property() public label?: string;
+
+  protected render() {
+    const parts = this.value?.split(":") || [];
+
+    return html`
+      <paper-time-input
+        .label=${this.label}
+        .hour=${parts[0] ?? "0"}
+        .min=${parts[1] ?? "00"}
+        .sec=${parts[2] ?? "00"}
+        .amPm=${false}
```

Comment:
We need to do this and set the format?

Reply:
Also can we check if the current locale is 24 or 12 hour based ?

Reply:
yes...

Reply:
```ts
const bla = new Date().toLocaleString();
const useAMPM = bla.includes('AM') || bla.includes('PM')
```